content/business-models/none.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "None"
+date: 2019-01-09
+weight: 1
+---
+
+This is typically the ur-form of sustainable Free and Open Source Communities.
```

Comment:
Ur-form is not that commonly used a term, something more accessible like original form, or starting point is easier to read.